apis/core/v1alpha2/core_types.go

```diff
@@ -221,6 +221,7 @@ type Revision struct {
 // +kubebuilder:resource:categories={crossplane,oam}
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:JSONPath=".spec.workload.kind",name=WORKLOAD-KIND,type=string
+// +kubebuilder:printcolumn:name="age",type="date",JSONPath=".metadata.creationTimestamp"
```

Comment:
is the timestamp same as AGE (which is the difference between NOW and the creationtime"

Reply:
It's create time, evaluation will automatically done by k8s